website/astro.config.ts

```diff
@@ -404,6 +410,11 @@ export default defineConfig({
 	adapter: netlifyStatic(),
 
 	vite: {
+		resolve: {
+			alias: {
+				"@": path.resolve(__dirname, "./src"),
```

Comment:
This also does the same thing

```suggestion
				"@": new URL("./src", import.meta.url)
```

Reply:
Hey, This is great we can use this also as this is much cleaner way without any nodejs module import like path etc, it will use the global URL obj, but there is one change with this we need to use This also does the same thing` "@": new URL("./src", import.meta.url).pathname` instead of only `"@": new URL("./src", import.meta.url)` otherwise it will returns the  URL object not the relative path. I made a new commit with this , Thanks @anonrig for the suggestion.